File: app/src/main/java/com/song/deviceinfo/ui/app/AppViewModel.java
package com.song.deviceinfo.ui.app;

import android.content.Context;

import com.song.deviceinfo.info.AppInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/9/22.
 */
public class AppViewModel extends NormalViewModel {

    public List<Pair<String, String>> getAppInfo(Context context) {
        return AppInfo.getAppInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/app/AppViewModel.kt
package com.song.deviceinfo.ui.app

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.AppInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/9/22.
 */
class AppViewModel : NormalViewModel() {
    fun getAppInfo(context: Context): List<Pair<String, String>> {
        return AppInfo.getAppInfo(context)
    }
}
